pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
 section: docs
 ---
 
-The release notes discuss the changes impacting users and implementers:
+### Introduction
 
+Just as your schemas evolve over time, the JSON Schema specification does as well. This guide provides useful information to make upgrading your schemas across versions of the specification a breeze.
+
+Learn how to use our migration guides and tooling ([AlterSchema](https://alterschema.sourcemeta.com/)) to bring your work with you.
+
+### Keywords Overview
+
+Here is a comprehensive overview to get you started.
+
+| All Keywords                | Specification | Draft introduction | Removed | Changed                                                                                      |
+| --------------------------- | ------------- | ------------------ | ------- | -------------------------------------------------------------------------------------------- |
+| `$anchor`                   | Core          | 2019-09            | No      | Partially replaced `$id`                                                                     |
+| `$comment`                  | Core          | 07                 | No      |                                                                                              |
+| `$id`                       | Core          | 06                 | No      | Replaced `id`                                                                                |
+| `$defs`                     | Core          | 2019-09            | No      | Replaced `definitions`                                                                       |
+| `$dynamicAnchor`            | Core          | 2020-12            | No      | Replaced `$recursiveAnchor`                                                                  |
+| `$dynamicRef`               | Core          | 2020-12            | No      | Replaced `$recursiveRef`                                                                     |
+| `$recursiveAnchor`          | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicAnchor`                                                                 |
+| `$recursiveRef`             | Core          | 2019-09            | 2020-12 | Replaced by `$dynamicRef`                                                                    |
+| `$ref` (legacy)             | Core          | 03                 | 2019-09 | Replaced by `$ref`                                                                           |
```

Comment:
This entry feels weird. We are saying that `$ref` was removed on `2019-09` and replaced by `$ref`, which feels very confusing. The change this specific entry should be documenting is that before 2019-09, `$ref` took precedence over any other sibling keyword. In other words, before 2019-09, if you put any other keywords alongside `$ref`, it would be ignored.